### DISCUSSION TITLE
I can't translate some parts of the UI (Spanish Language)

### DISCUSSION THREAD

**Discussion by laramaceo** [tnodir/fort#308] [General]
There are parts of the Fort Firewall UI that I cannot translate because they are not available in the Crowdin translation guide. I request the inclusion of these texts.
![Parts of the UI without translation ](https://github.com/user-attachments/assets/7230fb73-8b91-43e0-9223-7772bd2f82c4)

![Main](https://github.com/user-attachments/assets/4563b1d1-1f01-4fbc-91c7-2d0b10fb32cf)

How can I get the dark theme for Fort Firewall?

**Comment by tnodir** [comment]
> How can I get the dark theme for Fort Firewall?

Which version do you use, 32 or 64-bit?

https://github.com/tnodir/fort/wiki/FAQ#limitations-of-the-32-bit-version-of-fort-firewall

**Reply by laramaceo** [reply]
I still use an old 32 Bit PC.

**Comment by tnodir** [comment]
The "Main" group is the name of default app group.

You can rename it in the Options: App Groups.

**Comment by tnodir** [comment]
> I request the inclusion of these texts.

Done.
